modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/getItemField.ts

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
@markocikos Refactor function to do only one thing. Add some examples trying to clarify what is happening :sweat_smile:

Reply:
Much better, now with single returns! :clap: